### PR TITLE
Auto 452 add tag to client

### PR DESCRIPTION
**Pull request by ShannyWorkiz**

### CODE REVIEW

File: cypress/page-objects/Clients/allClientsPage.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import {ClientPage} from "./clientPage"
 import { CreateClientPage } from "./createClientPage";
+import {cli} from "cypress";
```
Comment (IdoAdler2) [inline cypress/page-objects/Clients/allClientsPage.ts]: @ShannyWorkiz ???

Comment (ShannyWorkiz) [reply cypress/page-objects/Clients/allClientsPage.ts]: @IdoAdler2 Just pushed a fix

File: cypress/page-objects/Clients/allClientsPage.ts [hunk 2]
```diff
@@ -25,4 +25,9 @@ export class AllClientsPage {
             cy.get('div').contains('Job ID').click();
         }
     }
+
+    isClientContainsTag(clientId: JQuery<HTMLElement>, tag: string)
+    {
+        cy.get('.rt-tr-group.pointer .rt-td', {timeout: 10000}).filter(`:contains("${clientId.toString()}")`).siblings('div').find('div').should('contain.text', tag);
```
Comment (IdoAdler2) [inline cypress/page-objects/Clients/allClientsPage.ts:31]: @ShannyWorkiz Why do we need the siblings? what does it do?

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -1,68 +1,65 @@
...
-      {
-        expect($el.text()).to.be.eq(textToFind);
-        return false;
-      }
-    else if (index == $list.length-1)
-    {
-      console.log(`these are all the texes were in the elements and didnt equal to text: ${allElementsText}`)
-      expect($el.text()).to.be.eq(textToFind);
-    }
-  })
```
Comment (IdoAdler2) [inline cypress/support/commands.ts:65]: @ShannyWorkiz Why did u delete the console log? it suppose to be there to help us?

File: cypress/page-objects/Clients/allClientsPage.ts (hunk 2 above)
Comment (ShannyWorkiz) [reply cypress/page-objects/Clients/allClientsPage.ts:31]: It selected a client by ID and then goes to its name (that is why using sibling- it the one after the ID)

File: cypress/page-objects/Clients/allClientsPage.ts (hunk 1 above)
Comment (IdoAdler2) [reply cypress/page-objects/Clients/allClientsPage.ts]: @ShannyWorkiz great. thanks for the response. 👌

File: cypress/page-objects/Clients/allClientsPage.ts (hunk 2 above)
Comment (IdoAdler2) [reply cypress/page-objects/Clients/allClientsPage.ts:31]: @ShannyWorkiz great. thanks for the response. 👌

**Pull request merged**